mopidy_tunein/__init__.py

```diff
@@ -20,6 +20,7 @@ def get_default_config(self):
     def get_config_schema(self):
         schema = super(Extension, self).get_config_schema()
         schema['timeout'] = config.Integer(minimum=0)
+        schema['filter'] = config.String(optional=True)
```

Comment:
Might as well get this validated properly:
```
config.String(optional=True, choices=('stations', 'shows'))
```

Reply:
thanks @kingosticks for the addition.